transformation/src/jade-schema/tables/human_donor.table.json

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "human_donor",
+  "columns": [
+    {
+      "name": "id",
+      "datatype": "string",
+      "type": "primary_key"
+    },
+    {
+      "name": "time_created",
+      "datatype": "timestamp",
+      "type": "required"
+    },
+    {
+      "name": "age",
+      "datatype": "string",
+      "type": "required"
+    },
+    {
+      "name": "ethnicities",
+      "datatype": "string",
+      "type": "repeated"
+    },
+    {
+      "name": "organism",
+      "datatype": "string",
+      "type": "required"
+    },
+    {
+      "name": "sex",
+      "datatype": "string",
+      "type": "required"
+    },
+    {
+      "name": "award",
+      "datatype": "string",
+      "type": "required"
+    },
+    {
+      "name": "lab",
+      "datatype": "string",
+      "type": "required"
+    },
+    {
+      "name": "life_stage",
+      "datatype": "string",
+      "type": "required"
+    },
+    {
+      "name": "parents",
+      "datatype": "string",
+      "type": "repeated"
```

Comment:
Should this include a `link` to `human_donor`?

Reply:
No, this is a URI to an Encode object because not all parents are necessarily donors